docs/reference_info_json.md

```diff
@@ -1,369 +1,696 @@
+        }
+        ```
+    * `center_point`
+        * The centroid (geometric center) of the LEDs. Used for certain effects.
+        * Default: `[112, 32]`
+    * `driver` (Required)
+        * The driver to use. Must be one of `aw20216`, `ckled2001`, `custom`, `is31fl3731`, `is31fl3733`, `is31fl3736`, `is31fl3737`, `is31fl3741`, `is31fl3742a`, `is31fl3743a`, `is31fl3745`, `is31fl3746a`, `ws2812`.
+    * `hue_steps`
+        * The number of hue adjustment steps.
+        * Default: `8`
+    * `layout` (Required)
+        * List of LED configuration dictionaries. Each dictionary contains:
+            * `flags` (Required)
+                * A bitfield of flags describing the type of LED.
+            * `x` (Required)
+                * The position of the LED in the horizontal axis, from 0 to 224.
+            * `y` (Required)
+                * The position of the LED in the vertical axis, from 0 to 64.
+            * `matrix`
+                * The key matrix position associated with the LED.
+                * Example: `[0, 2]`
+            * Example: `{"matrix": [2, 1], "x": 20, "y": 48, "flags": 2}`
+    * `max_brightness`
+        * The maximum value which the HSV "V" component is scaled to, from 0 to 255.
+        * Default: `255`
+    * `sat_steps`
+        * The number of saturation adjustment steps.
+        * Default: `16`
+    * `speed_steps`
+        * The number of speed adjustment steps.
+        * Default: `16`
+    * `split_count`
+        * For split keyboards, the number of LEDs on each half.
+        * Example: `[16, 16]`
+    * `timeout`
+        * The LED activity timeout in milliseconds.
+        * Default: `0` (no timeout)
+    * `val_steps`
+        * The number of brightness adjustment steps.
+        * Default: `16`
+
+## Secure :id=secure
+
+Configures the [Secure](feature_secure.md) feature.
+
+* `secure`
+    * `enabled`
+        * Enable the Secure feature.
+        * Default: `false`
+    * `idle_timeout`
+        * Timeout while unlocked before returning to the locked state. Set to `0` to disable.
+        * Default: `60000` (1 minute)
+    * `unlock_sequence`
+        * A list of up to five matrix locations comprising the "unlock sequence".
+        * Example: `[[0, 0], [0, 1], [4, 3]]`
+    * `unlock_timeout`
+        * Timeout for the user to perform the unlock sequence. Set to `0` to disable.
+        * Default: `5000` (5 seconds)
+
+## Split Keyboard :id=split-keyboard
+
+Configures the [Split Keyboard](feature_split_keyboard.md) feature.
 
-Example:
+* `split`
+    * `bootmagic`
+        * `matrix`
+            * See [Bootmagic](#bootmagic) config.
+    * `enabled`
+        * Enable the Split Keyboard feature.
+        * Default: `false`
+    * `encoder`
+        * `right`
+            * `rotary`
+                * See [Encoder](#encoder) config.
+    * `main`
+        * How to determine the main/primary half. Must be one of `eeprom`, `left`, `matrix_grid`, `pin`, `right`.
```

Comment:
Probably not worth documenting this given its broken, <https://github.com/qmk/qmk_firmware/pull/18254>. Same goes for some of the other fields that PR flags issue to, matrix_grid.